Whiskey/en-US/about_Whiskey_DotNetBuild_Task.help.txt

```diff
@@ -0,0 +1,63 @@
+TOPIC
+    about_Whiskey_DotNetBuild_Task
+
+SUMMARY
+    Builds .NET Core SDK projects.
+
+DESCRIPTION
+    The `DotNetBuild` tasks runs the `dotnet.exe build` command for building .NET projects targeting .NET Core and .NET Standard. Pass a list of solutions files or .NET Core project files to the `Path` property. If no files are provided to `Path`, then the .NET Core SDK will search for any solution or project files in the working directory and build those. If the `dotnet.exe build` command returns a non-zero exit code the build will fail.
+
+    The task will automatically apply the following switches to the `dotnet.exe build` command and are not modifiable:
+    
+        * `--configuration=(Debug|Release)`: `Debug` when build run by a developer and `Release` when build run by a build server.
+        * `-p:Version=$(WHISKEY_VERSION_SEMVER1)`: the semver version 1 version for the current build.
+

```

Comment:
Document that the task and why the task sets the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` and `DOTNET_CLI_TELEMETRY_OPTOUT` environment variables on the build server. Also, should we only do this on the build server? Why are you setting these?

Reply:
`DOTNET_CLI_TELEMETRY_OPTOUT` prevents the following message:
```
Welcome to .NET Core!
---------------------
Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.

Telemetry
--------------
The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
```
`DOTNET_SKIP_FIRST_TIME_EXPERIENCE` prevents caching nuget packages to local package cache on the machine.

I'm not sure, should these be set on a build server? Or do we not care about these at all?

Reply:
I don't think we care about either of those things. If it output, it outputs. Hopefully, the telemetry output only shows once.

Caching is fine, so I don't think we need `DOTNET_SKIP_FIRST_TIME_EXPERIENCE`